src/python_routing_v02/build_tests.py

```diff
@@ -160,13 +157,15 @@ def build_test_parameters(
     )
 
 
-def parity_check(parity_parameters, run_parameters, nts, dt, results):
-    
+def parity_check(parity_parameters, run_parameters, results):
+    nts = run_parameters["nts"]
+    dt = run_parameters["dt"]
+
```

Comment:
These three lines (the `parity_check` function and the two variables below that) seem to have been dropped from the merge or were stomped on by a later commit.
@jhrehanoaa